src/Player.js

```diff
@@ -405,6 +405,20 @@ PlayerObject.prototype.getHomeComputer = function() {
     return AllServers[this.homeComputer];
 }
 
+PlayerObject.prototype.getUpgradeHomeRamCost = function() {
+    //Calculate how many times ram has been upgraded (doubled)
+    const currentRam = Player.getHomeComputer().maxRam;
+    const numUpgrades = Math.log2(currentRam);
+
+    //Calculate cost
+    //Have cost increase by some percentage each time RAM has been upgraded
+    const mult = Math.pow(1.58, numUpgrades);
+    var cost = currentRam * CONSTANTS.BaseCostFor1GBOfRamHome;
```

Comment:
why not combine this line with the next?

Reply:
I can, doesn't change much though